regression/scripts/sizing_loop/sizing_loop.py

```diff
@@ -0,0 +1,323 @@
+    freestream  = atmosphere.compute_values(altitude)
+    freestream0 = atmosphere.compute_values(6000.*Units.ft)  #cabin altitude
+    
+    
+    diff_pressure                  = np.max(freestream0.pressure-freestream.pressure,0)
+    fuselage                       = base.fuselages['fuselage']
+    fuselage.differential_pressure = diff_pressure 
+    
+    #now size engine
+    mach_number        = air_speed/freestream.speed_of_sound
+    
+    #now add to freestream data object
+    freestream.velocity    = air_speed
+    freestream.mach_number = mach_number
+    freestream.gravity     = 9.81
+    
+    conditions             = SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics()   #assign conditions in form for propulsor sizing
+    conditions.freestream  = freestream
+    
+    nose_load_fraction     = .06
+   
+    #now evaluate all of the vehicle configurations
+    for config in configs:
+        config.wings.horizontal_stabilizer.areas.reference = (26.0/92.0)*config.wings.main_wing.areas.reference
+         
+         
+        for wing in config.wings:
+            
+            wing = SUAVE.Methods.Geometry.Two_Dimensional.Planform.wing_planform(wing)
+            
+            wing.areas.exposed  = 0.8 * wing.areas.wetted
+            wing.areas.affected = 0.6 * wing.areas.reference
+            
+
+
+        fuselage                       = config.fuselages['fuselage']
+        fuselage.differential_pressure = diff_pressure 
+     
+        #now evealute weights
+
+        # diff the new data
+        
+        config.mass_properties.max_takeoff     = m_guess #take in parameters
+        config.mass_properties.takeoff         = m_guess 
+        config.mass_properties.max_zero_fuel   = base.mass_properties.max_zero_fuel
+        config.store_diff()
+       
+       
+    #now evaluate the weights   
+    weights = analyses.base.weights.evaluate() #base.weights.evaluate()  
+    #update zfw
+    empty_weight       = base.mass_properties.operating_empty
+    payload            = base.mass_properties.max_payload
+    zfw                = empty_weight + payload 
+    
+    base.max_zero_fuel = zfw  
+    base.store_diff()
+    for config in configs:
+        config.pull_base()
+    # ------------------------------------------------------------------
+    #   Landing Configuration
+    # ------------------------------------------------------------------
+    landing = nexus.vehicle_configurations.landing
+    landing_conditions = Data()
+    landing_conditions.freestream = Data()
+
+    # landing weight
+    landing.mass_properties.landing = base.mass_properties.max_zero_fuel
+    
+    # Landing CL_max
+    altitude                                         = nexus.missions.base.segments[-1].altitude_end
+    atmosphere                                       = SUAVE.Analyses.Atmospheric.US_Standard_1976()
+    p, T, rho, a, mu                                 = atmosphere.compute_values(altitude)
```

Comment:
This kind of unpacking no longer works. Need to check for other instances of this. It will blow up regressions depending on the machine your running.